avm/res/api-management/service/workspace/diagnostic/main.bicep

```
@@ -0,0 +1,100 @@
+metadata name = 'API Management Workspace Diagnostics'
+metadata description = 'This module deploys a Diagnostic at API Management Workspace scope.'
+
+@description('Conditional. The name of the parent API Management service. Required if the template is used in a standalone deployment.')
+param apiManagementServiceName string
+
+@description('Conditional. The name of the parent Workspace. Required if the template is used in a standalone deployment.')
+param workspaceName string
+
+@description('Required. Diagnostic Name.')
+param name string
+
+@description('Required. Logger resource ID.')
+param loggerId string
```

```suggestion
param loggerResourceId string
```
Note: Has implications in other places